code/products/mysql/connect-using-mysqlx.py

```diff
@@ -0,0 +1,28 @@
+import mysqlx
+
+# replace the variables accordingly
+SERVICE_URI = "aiven-service-URI"
```

Comment:
I'm not sure why wee keep these. If used with the variables this won't work since we also put data into the brackets. I think either these three lines should be removed or brackets below.

Reply:
If we decide to keep there three lines, we need to specify that aiven-service-URI is URI for mysqlx

Reply:
solved, i removed it :)